src/components/Filters/ViewsMenu.tsx

```diff
@@ -103,64 +113,68 @@ class ViewsMenu extends React.Component<ViewsMenuProps, ViewsMenuState> {
 
 		return (
 			<Wrapper>
-				<DropDownButton
-					ml={soloRender ? 0 : 30}
-					disabled={this.props.disabled}
-					quartenary={!this.props.dark}
-					light={this.props.dark}
-					outline
-					joined
-					alignRight={!soloRender}
-					noListFormat
-					icon={<FontAwesomeIcon icon={faChartPie} />}
-					label="Views"
-					{...this.props.buttonProps}
-				>
-					<Box py={1}>
-						{!hasViews && (
-							<Box py={2} px={3}>
-								{"You haven't created any views yet"}
+				<UseBreakpointWrapper values={compact}>
```

Comment:
We don't need a wrapper component. Since `useBreakpoint` is a hook, we should use it as such.
We can either:
1. Refactor this component to make it a functional one, and use the hook as usual.
2. Extract the render body (or the one that makes sense) to another functional component, and use the hook there as usual.

I would prefer 1. as it is a very simple class component and should be pretty trivial to refactor it.

Reply:
Actually now that I look at the code, the hook shouldn't even live here, the parent should use the hook, and then just pass the result from the hook (something like `shouldShowLabel`), instead of the entire `compact` array.